### PR TITLE
Client

### PR DESCRIPTION
**Pull request by ben-cunningham**

### CODE REVIEW

File: tests/test_http_client.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            body='{ \
+                "data" : [1,2,3] \
+            }')
+
+        def completion(payload, error):
+            assert payload['data'] == [1,2,3]
+            assert payload['data'] != [3,2,1]
+
+        client = HttpClient()
+        client.submit_request('/users/123',
```
Comment (ahendy0) [inline tests/test_http_client.py]: Only newline separate if it deserves it: EG

if one parameter is more important than the others or the line is longer than 79 chars

**Comment by ahendy0** [comment]
how do i approve a PR?

**Comment by ben-cunningham** [comment]
@ahendy0 Generally when the assignee think's it's good to go, then you merge + delete the branch

**Comment by ben-cunningham** [comment]
@ahendy0 fixed jah hooligan

**Pull request merged**